### DISCUSSION TITLE
Considering redesign

### DISCUSSION THREAD

**Discussion by lynchaj** [lynchaj/nhyodyne#55] [Announcements]
Hi
I added a folder in the nhyodyne GitHub for files related to discussion of a redesign of the system to address some of its shortcomings.  This should be considered experimental with the plan being to reuse the nhyodyne circuitry and software but in a new, larger format.  The goal is to have a more expansive bus and slightly higher integration resulting in fewer boards per system using overall fewer parts.  Essentially combinations of already paired boards such as processor & clock, ROM & RAM, etc.  The approach differs from earlier work in an attempt to design for a board cage/chassis from the start rather than trying to retrofit one later.

Thanks, Andrew Lynch

**Comment by lynchaj** [comment]
Hi
Most of the redesign discussion is taking place by email off-list.  If you are interested in the discussion please send me your email and I'll include you.  Plus you can respond here.  Here is the folder with the redesign files

https://github.com/lynchaj/nhyodyne/tree/main/!REDESIGN

Thanks, Andrew Lynch

**Comment by lynchaj** [comment]
Hi,
Work continues and have done some major updates.  I removed the old files snapshot and reposted a new one which can be found here

https://github.com/lynchaj/nhyodyne/tree/main/!REDESIGN

Goal is to update the nhyodyne bus specification to be more general and better facilitate other hobbyist projects such as Z180, 6502, 6809, 8088, 8086, TMS9900, 68K, etc.  Of course, is still fairly Z80 centric but providing other bus signals and voltages can make this a lot easier than it has been.

The backplane supports BAI/BAO and IEI/IEO chaining with jumpers built in.  Started some initial trace routing on the backplane to make sure my assumptions regarding power distribution and the chaining are correct.

Thanks, Andrew Lynch

**Comment by lynchaj** [comment]
Hi
Not sure who can see these discussion notes and are actually reaching anyone.  If you can read this, please do a quick reply.

Thanks, Andrew Lynch

**Reply by wwarthen** [reply]
> Hi Not sure who can see these discussion notes and are actually reaching anyone. If you can read this, please do a quick reply.

I'm am seeing these posts and following.  I get notified when you post because I am "watching" the GitHub repository.

I have not replied yet because I need some time to digest the new format -- I'm not as fast as some folks 😀.  I guess my initial reaction is that it seems very comprehensive.  As a result it is also a pretty large form factor (about the same as S100?).  Any thoughts on enclosure?

Thanks,

Wayne

**Comment by lynchaj** [comment]
Hi Wayne
Thanks!  Yes, still mulling an enclosure.  I think it will have to contain a board cage, power supply, and two 3.5" drives (floppy or hard drive) so that's a pretty big enclosure.  Mostly working on the board set and board cage now though.  I am thinking one of those Meanwell power supplies with GND, 5V, +12V, and -12V rails.  Even getting a matched set of boards and board cage would be a dramatic improvement over present nhyodyne though.  My systems so far have all been "pile of wires" too delicate to move.  A board cage would at least reduce the number of pieces to move to something manageable.
Form factor is 225x150 cm for components not counting the board guide and bus connector exclusion zones.  Roughly the same area as S-100 maybe a little smaller but no power regulators either so about the same overall.  
I am sizing something that could theoretically fit a uPD7220 GDC video board project and that's just not feasible with present nhyodyne format.  It would have to be 3 or 4 boards deep stack to fit all the circuitry.  I think uPD7220 GDC is representative of the top end of hobbyist projects as size goes.  Most are smaller
Thanks for responding, much appreciated.  Andrew Lynch




    On Thursday, February 16, 2023 at 01:36:10 PM EST, Wayne Warthen ***@***.***> wrote:  
 
 



Hi Not sure who can see these discussion notes and are actually reaching anyone. If you can read this, please do a quick reply.


I'm am seeing these posts and following. I get notified when you post because I am "watching" the GitHub repository.

I have not replied yet because I need some time to digest the new format -- I'm not as fast as some folks 😀 . I guess my initial reaction is that it seems very comprehensive. As a result it is also a pretty large form factor (about the same as S100?). Any thoughts on enclosure?

Thanks,

Wayne
